### DISCUSSION TITLE
[Feature Request] OpenWeather API Language

### DISCUSSION THREAD

**Discussion by fdbailes** [daledavies/jump#78] [Feature Requests]
Hi there, can you please add environment option to set openweather api lang for descriptions. 

Parameter is there on the API (https://openweathermap.org/current)

It might be great to have the option for this awesome app.

Thanks.

**Comment by daledavies** [comment]
Thanks for reaching out.

I was thinking the other day about adding translations for the few strings that are used in jump, this suggestion ties nicely into that so I'll keep it in mind.

**Comment by fdbailes** [comment]
Yes, that is a great idea.

I was about to raise another issue to present the idea of custom greeting variable, because language is fixed in the current one.

But maybe is a good idea to let it be custom. One less variable to translate.

Will look forward for the changes.

Thanks

**Comment by daledavies** [comment]
Would you just want a fixed greeting, not one that changes with the time of day?

**Comment by fdbailes** [comment]
In my case I'm using it as a title/description, but you are right if it is a greeting it should be dynamic.

Maybe it can serve both purposes. Just thinking out loud.

**Comment by daledavies** [comment]
It could be extended to allow a dynamic greeting, or a single user specified string. Again I'll keep it in mind for the future, thanks 😀

**Comment by daledavies** [comment]
Custom greeting message implemented in commit https://github.com/daledavies/jump/commit/ed9178bf8e91d7584ccc6ce9548b1ff94821c084

**Comment by daledavies** [comment]
Translations and corresponding OpenWeather API language implemented in commit https://github.com/daledavies/jump/commit/0c2c320c7574265bba02e793b8c00f176f7de4fe

**Comment by daledavies** [comment]
Will package these up in the next releae :)
